src/Sylius/Bundle/ApiBundle/Resources/config/serialization/Order.xml

```diff
@@ -44,12 +39,13 @@
             <group>sylius:admin:order:index</group>
             <group>sylius:admin:order:show</group>
             <group>sylius:shop:order:index</group>
-            <group>sylius:shop:order:show</group>
+            <group>sylius:shop:cart:show</group>
         </attribute>
 
         <attribute name="customer">
             <group>sylius:admin:order:index</group>
             <group>sylius:admin:order:show</group>
+            <group>sylius:shop:cart:show</group>
```

Comment:
Why did you add this group?

Reply:
For the sake of API consistency, we can now clearly see when the customer is set and when they are not.

Reply:
But it is a shop endpoint, so there we should always get "ourselves". IMO revert this change as it is not needed by default, if anyone would like to add it in the end app, it would be quite easy to customize

Reply:
We chose to embed the `[customer][email]` field here, or show `null` if no customer is set.